Bluetooth: controller: Remove is_enabled flag for BIS events

cvinayak opened the pull request:
Remove is_enabled flag in LLL of BIS events context, as the
status returned from ticker_stop interface is sufficient to
determine if the Broadcast ISO Sync was established or not.
HCI Command Disallow as reason is to be returned if sync was
not established or sync was loss prior to call of HCI LE
BIG Terminate Command was called.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>

cvinayak committed:
Bluetooth: controller: Remove is_enabled flag for BIS events

Remove is_enabled flag in LLL of BIS events context, as the
status returned from ticker_stop interface is sufficient to
determine if the Broadcast ISO Sync was established or not.
HCI Command Disallow as reason is to be returned if sync was
not established or sync was loss prior to call of HCI LE
BIG Terminate Command was called.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>

pabigot reviewed (changes requested):
Yes, this does fix the build error.  But please do the move of `ll_sync_iso.h` from the Nordic-specific include path to the general include path in a separate commit, rather than incorporating it into something that removes a flag and says nothing about the move.

An explanation for why the flag was removed would be nice too.

cvinayak committed:
Bluetooth: controller: Move lll_sync_iso.h common to all vendors

Move lll_sync_iso.h as common to all vendors, as currently
there is nothing that is vendor specific.

Fixes #31044.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>

cvinayak commented:
@Thalley this PR is breaking CI, could you prioritize review of this PR?
